src/app/container/launch/launch.component.ts

```diff
@@ -72,14 +72,15 @@ export class LaunchComponent extends Base implements OnInit, OnChanges {
       .subscribe((descriptors: Array<Workflow.DescriptorTypeEnum>) => {
         this.descriptors = descriptors;
         this.filteredDescriptors = this.filterDescriptors(this.descriptors, this._selectedVersion, this.versionsFileTypes);
```

Comment:
After a quick skim of the code, I would be inclined to put this line inside `reactToDescriptor`, just to avoid "everytime you call `reactToDescriptor`, make sure to first call `this.filterDescriptors`". 

But "everywhere" is only twice, and there may be a subtlety I'm missing, so your call.

Reply:
Thanks Charles for the suggestion. I tried with it, but interestingly enough the page did not get updated correctly. I'm actually very curious as to why, if anyone knows please let me know. But I think we will leave the code like that for now